commit 44436c569641446212578d3f0e27490c75dac0c4
Author: Rob Johnston <rob.johnston@joyent.com>
Date:   2019-05-02T22:24:25+00:00 (5 months ago)
    
    OS-7485 typo in SSG-2028R-ACR24L and JSP-7001 slot maps
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Joshua M. Clulow <josh@sysmgr.org>

diff --git a/usr/src/lib/fm/topo/maps/Joyent,Joyent-Storage-Platform-7001/Joyent-Storage-Platform-7001-slot-hc-topology.xml b/usr/src/lib/fm/topo/maps/Joyent,Joyent-Storage-Platform-7001/Joyent-Storage-Platform-7001-slot-hc-topology.xml
index 6b16063427..9362ae3e8d 100644
--- a/usr/src/lib/fm/topo/maps/Joyent,Joyent-Storage-Platform-7001/Joyent-Storage-Platform-7001-slot-hc-topology.xml
+++ b/usr/src/lib/fm/topo/maps/Joyent,Joyent-Storage-Platform-7001/Joyent-Storage-Platform-7001-slot-hc-topology.xml
@@ -11,7 +11,7 @@
  source.  A copy of the CDDL is also available via the Internet at
  http://www.illumos.org/license/CDDL.
 
- Copyright (c) 2018, Joyent, Inc.
+ Copyright (c) 2019, Joyent, Inc.
 
 -->
 
@@ -201,7 +201,7 @@
              <propgroup name='ipmi' version='1'
                name-stability='Private' data-stability='Private' >
                <propval name='entity-list' type='string_array' >
-                 <propitem value='P2-DIMMF1 Temp' />
+                 <propitem value='P2-DIMMF2 Temp' />
                </propval>
              </propgroup>
            </node>
diff --git a/usr/src/lib/fm/topo/maps/SMCI,SSG-2028R-ACR24L/SSG-2028R-ACR24L-slot-hc-topology.xml b/usr/src/lib/fm/topo/maps/SMCI,SSG-2028R-ACR24L/SSG-2028R-ACR24L-slot-hc-topology.xml
index 6b16063427..9362ae3e8d 100644
--- a/usr/src/lib/fm/topo/maps/SMCI,SSG-2028R-ACR24L/SSG-2028R-ACR24L-slot-hc-topology.xml
+++ b/usr/src/lib/fm/topo/maps/SMCI,SSG-2028R-ACR24L/SSG-2028R-ACR24L-slot-hc-topology.xml
@@ -11,7 +11,7 @@
  source.  A copy of the CDDL is also available via the Internet at
  http://www.illumos.org/license/CDDL.
 
- Copyright (c) 2018, Joyent, Inc.
+ Copyright (c) 2019, Joyent, Inc.
 
 -->
 
@@ -201,7 +201,7 @@
              <propgroup name='ipmi' version='1'
                name-stability='Private' data-stability='Private' >
                <propval name='entity-list' type='string_array' >
-                 <propitem value='P2-DIMMF1 Temp' />
+                 <propitem value='P2-DIMMF2 Temp' />
                </propval>
              </propgroup>
            </node>
